examples/blas/compile_time_dispatching/level3/gemm_usm_mklcpu_cublas.cpp

```diff
@@ -162,9 +165,9 @@ int run_gemm_example(const sycl::device &cpu_dev, const sycl::device &gpu_dev) {
     if (!gpu_A || !gpu_B || !gpu_C ) {
         throw std::runtime_error("Failed to allocate USM memory.");
     }
-    gpu_queue.memcpy(gpu_A, A, sizea * sizeof(float)).wait();
-    gpu_queue.memcpy(gpu_B, B, sizeb * sizeof(float)).wait();
-    gpu_queue.memcpy(gpu_C, C, sizec * sizeof(float)).wait();
+    gpu_queue.memcpy(gpu_A, A.data(), sizea * sizeof(float)).wait();
```

Comment:
wait_and_throw()?

Reply:
Changed to wait_and_throw() for memcpy calls